### ISSUE TITLE
Avare Automatic Route Planning 

### ISSUE THREAD

**Issue by Rosedog09** [apps4av/avare#472]
It would be helpful if the Avare app would automatically find a preferred route (Similar to Skyvector) for IFR flights instead of it all being done manually. 

https://groups.google.com/g/apps4av-forum/c/1ShnH76FqEU

**Comment by gustafson** [comment]
Jeff and I talked about whether this should be faa preferred routes vs recently assigned routes (which some commercial vendors provide).

For now I think it should be faa preferred because:

1. FAA publishes on a cadence
2. Its official from an official source
3. It is comparatively static and can live in the app without requiring an active internet connection
4. It requires no additional resources on our end

I can see how both would be useful, but I don't have a resource for recently assigned.  If anybody has some suggestion let me know.

**Comment by gustafson** [comment]
The chart issue supporting this request is [here](https://github.com/gustafson/avare-charts/issues/4)

**Comment by Rosedog09** [comment]
I would agree with FAA preferred over anything else, but it is possible (again I dont know how this works so it maybe too much work) you could add an option for the pilot to choose one way or the other.
